mantidproject/mantid
# IndirectCommon - Improve CheckAnalysers error message

CheckAnalysers now throws a RuntimeError if the analyser can not be found rather than a generic error being raised.
# To test
- Check the tests for IndirectCommon pass
- Run the following script:

```
from IndirectCommon import CheckAnalysers

data = "1,2,3,4,5"
ws = CreateWorkspace(DataX=data, DataY=data)
ws2 = CreateWorkspace(DataX=data, DataY=data)
CheckAnalysers(ws.getName(), ws2.getName())
```
- Observed the error improved error message in the results log

Fixes #15462

**Does not need to be in the release notes**

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
